sui_programmability/examples/utils/tests/safe_tests.move

```diff
@@ -66,21 +66,20 @@ module utils::safe_tests {
         let initial_funds = 1000u64;
         create_safe(scenario, owner, initial_funds);
 
-        ts::next_tx(scenario, owner);
-        let mut safe = ts::take_shared<Safe<SUI>>(scenario);
-        let cap = ts::take_from_sender<OwnerCapability<SUI>>(scenario);
+        scenario.next_tx(owner);
+        let mut safe = scenario.take_shared<Safe<SUI>>();
+        let cap = scenario.take_from_sender<OwnerCapability<SUI>>();
 
-        safe::withdraw(&mut safe, &cap, initial_funds, ts::ctx(scenario));
-        ts::next_tx(scenario, owner);
-        let withdrawn_coin = ts::take_from_sender<Coin<SUI>>(scenario);
-        assert!(coin::value(&withdrawn_coin) == initial_funds, 0);
+        safe.withdraw(&cap, initial_funds, ts::ctx(scenario));
+        scenario.next_tx(owner);
+        let withdrawn_coin = scenario.take_from_sender<Coin<SUI>>();
+        assert!(withdrawn_coin.value() == initial_funds, 0);
 
         test_utils::destroy(withdrawn_coin);
-        ts::return_to_sender(scenario, cap);
+        scenario.return_to_sender(cap);
         ts::return_shared(safe);
 
-
-        ts::end(scenario_val);
+        scenario_val.end();
```

Comment:
With the introduction of receiver syntax, we no longer need to create a different binding for the `&mut scenario` vs taking the scenario by value -- can you replace:

```move
let mut scenario_val = ts::begin(TEST_SENDER_ADDR);
let scenario = &mut scenario_val;
```

With just:

```move
let mut scenario = ts::begin(TEST_SENDER_ADDR);
```

There may be some other tests that need similar treatment.